palvelutarjotin/urls.py

```diff
@@ -38,6 +40,21 @@ def readiness(*args, **kwargs):
     return HttpResponse(status=200)
 
 
-urlpatterns += [path("healthz", healthz), path("readiness", readiness)]
+def version(*args, **kwargs):
```

Comment:
Getting rid of the "Make sure allowing safe and unsafe HTTP methods is safe here." security [hotspot](https://sonarcloud.io/project/security_hotspots?id=City-of-Helsinki_kultus-api&pullRequest=321&resolved=false&types=SECURITY_HOTSPOT&tab=how_to_fix) can be done with:

```patch
diff --git a/palvelutarjotin/urls.py b/palvelutarjotin/urls.py
index 8938234..a83857c 100644
--- a/palvelutarjotin/urls.py
+++ b/palvelutarjotin/urls.py
@@ -5,6 +5,7 @@ from django.http import HttpResponse
 from django.urls import include, path
 from django.utils.translation import ugettext
 from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_GET
 from helusers.admin_site import admin

 from common.utils import get_api_version
@@ -40,6 +41,7 @@ def readiness(*args, **kwargs):
     return HttpResponse(status=200)


+@require_GET
 def version(*args, **kwargs):
     response_json = {
         "status": "ok",
```

Reply:
fixed on #323